### ISSUE TITLE
No inline message for expect when using mocha and chai

### ISSUE THREAD

**Issue by sh33dafi** [wallabyjs/public#549]
### Issue description or question
When running this test 
```javascript
import {expect} from "chai";
describe("Foo", () => {
    it("bleh", () => {
        expect("foo").to.equal("food");
    });
});
```
I expect an inline message "expected 'foo' to equal 'food'" but the message is not shown in the editor

The test fails, the line is marked as failure and the message is shown in the "Failing Tests" tab 
<img width="611" alt="screen shot 2016-04-06 at 23 05 29" src="https://cloud.githubusercontent.com/assets/6681036/14332893/1ab96c58-fc4c-11e5-8c9e-3e66c7681d5a.png">

Full code can be found [here](https://gist.github.com/sh33dafi/0aa16686fd3dc0bbb0786cd5b378d091) `npm run init` to install dependencies

### Wallaby.js configuration file

```javascript
var babel = require('babel-core');
var wallabyWebpack = require('wallaby-webpack');
var webpack = require('webpack');

var webpackPostprocessor = wallabyWebpack({
  module: {
    loaders: []
  }
});

module.exports = function () {
  return {
    files: [
      {pattern: 'src/**/*.ts', load: false},
      {pattern: 'test/**/*.spec.ts', ignore: true}
    ],

    tests: [
      {pattern: 'test/**/*.spec.ts', load: false},
      {pattern: 'node_modules/**/*.js', ignore: true}
    ],

    preprocessors: {
      '**/*.js': file => babel.transform(file.content, {sourceMap: true})
    },
    testFramework: "mocha",
    postprocessor: webpackPostprocessor,
    bootstrap: function () {
      window.__moduleBundler.loadTests();
    },
    env: {
      runner: require('phantomjs-prebuilt').path,
      params: { runner: '--web-security=false' }
    }
  };
};
```

### Code editor or IDE name and version

WebStorm 2016.1

### OS name and version
OSX

**Comment by ArtemGovorov** [comment]
Confirmed, the issue seems to be with PhantomJs 2.x. If you remove the `env` setting (so it starts using PhantomJs 1.9.8), then it works as expected. 

Will be investigating why it happens. As a workaround in a meantime, you may switch to PhantomJs 1.9.8 by just commenting out the `env` setting (if it's possible for your project).

**Comment by ArtemGovorov** [comment]
The issue is fixed in core v1.0.213. The core will be updated automatically in a few minutes after your next wallaby.js restart. To get the update ASAP, you may restart your IDE and start wallaby.

**Comment by sh33dafi** [comment]
Thanks for the info!
